src/components/IndividualArticle.jsx

```diff
@@ -0,0 +1,48 @@
+import LoadingScreen from './Screens';
+import { useState, useEffect } from 'react';
+import { getSingleArticle } from '../utils/apis';
+import { useParams } from 'react-router-dom';
+
+export default function ArticleView() {
+  const [currentArticle, setCurrentArticle] = useState({});
+  const [isLoading, setIsLoading] = useState(false);
+
+  const id = useParams();
```

Comment:
id should be destructured off the params object that is returned from the useParams